### DISCUSSION TITLE
Smart links and selection of several countries

### DISCUSSION THREAD

**Discussion by juanctr3** [giuliopons/adadmin#80] [Ideas]
Hello. I like your development, but I would like you to have two more options that I leave you as a suggestion.
1. The ability to select several countries at once for banner advertising.
2. It would be great to be able to generate links in a similar way to a shortener, but that the system could count each click. These links can be placed in the descriptions of Youtube videos or in forums, on pages, etc (links only) and advertisers can pay for each click.

If you can do that development only for me, please tell me and I'll pay you.

**Comment by giuliopons** [comment]
Hi,
I have to analyze better the first request because without any doubt it requires a lot of code (also changes to DB and handle old installations), and probably a small budget could help (please, write me privately at giulio@barattalo.it). 
I'm already working on the second point, because all you need is to retrieve the **ClickTag**, which at this moment is visible only in HTML5 banners (since you have to pick it and place it in your HTML code).

The next update (4.0.4) will include this pont 2 and also an encryption key to prevent fraudulent users that try to manually change the URL.

**Comment by giuliopons** [comment]
The point 2 of your request is available.
Just click on the link "Show clicktag" in the banner details, to show the complete link of the click tag. You can use that link in articles or post, and it will track each time that link is clicked.

![image](https://github.com/giuliopons/adadmin/assets/1871627/1058d447-5c77-4811-bd1a-5a838939c4c1)

**Comment by giuliopons** [comment]
I've also added a new feature in v.4.1.0 which lets you retrieve a "Position link" which is a "clicktag" for all the banners ina position. By using that link when a user click on that link it will be redirected to the next banner in that position.
